cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -0,0 +1,119 @@
+import frappe
+
+def validate(self, method):
+    # Call the function to insert or update custom fields
+    insert_custom_fields(self)
+    salary_component(self)
+
+def insert_custom_fields(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `insert_custom_fields`

Add return type annotation: `None`

(ANN201)